stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/components/avatar/GroupAvatar.kt

```diff
@@ -80,15 +81,16 @@ public fun GroupAvatar(
                 .fillMaxHeight(),
         ) {
             for (imageIndex in 0 until imageCount step 2) {
+                val user = avatarUsers[imageIndex]
                 if (imageIndex < imageCount) {
-                    UserAvatar(
+                    Avatar(
```

Comment:
Same here.
Shouldn't we use `ChatTheme.componentFactory.UserAvatar()`?? :thinking:

Reply:
Answered above 🙂